src/state/deposit.ts

```diff
@@ -0,0 +1,34 @@
+import { isMainnet } from '@/constants/networks';
+import type { PayloadAction } from '@reduxjs/toolkit';
+import { createSlice } from '@reduxjs/toolkit';
+
+
+export enum DepositType {
+  Funkit = 'funkit',
+  Standard = 'standard',
+}
+
+export interface DepositState {
+  depositType: DepositType;
+}
+
+const initialState: DepositState = {
+  depositType: isMainnet ? DepositType.Funkit : DepositType.Standard,
+};
+
+export const depositSlice = createSlice({
```

Comment:
Thanks for iterating!!
I was actually thinking we could leverage `DepositDialogProps` and just pass in the `depositType` to the dialog via props. Let me know if that is do-able!